### PR TITLE
Regex and test for credit card numbers

### LINKED ISSUE #83: Create regex to detect credit card numbers

### PR DESCRIPTION
**Pull request by alexisc0305**
Develop regex and test to detect credit card numbers.
Closes #83

### CODE REVIEW

**Review by claesmk — CHANGES_REQUESTED** [review]
See commend on handling dashes in credit card numbers.  Also note that [0-9] could be simplified with \d

File: test_team2.py [hunk 1]
```diff
@@ -30,6 +31,21 @@ def test_us_ssn(self):
 
         ssn = 'SSN: 123-456-7890'
         self.assertFalse(find_us_ssn(ssn))
+    
+    def test_credit_card_number(self):
+        # Test valid credit card number
+        ccn = '4444 4444 4444 4444' # Visa, Mastercard, Discover
+        self.assertTrue(find_credit_card_number(ccn))
+
+        ccn = '3333 333333 33333' # American Express
```
[the remark this answers is not in the document]
Comment (alexisc0305) [reply test_team2.py:42]: Thank you for your feedback!

**Review by scary-monsters — APPROVED** [review]
Awesome. Nice job.

**Review by scary-monsters — APPROVED** [review]

**Review by Drdj11 — COMMENTED** [review]
clean regex

**Review by jscedeno — COMMENTED** [review]
looks good

**Review by claesmk — APPROVED** [review]
Looks good

**Pull request merged**